pom.xml

```diff
@@ -212,5 +219,28 @@
             <version>${junit.version}</version> <!-- Or whatever JUnit you're using. -->
             <scope>test</scope>
 		</dependency>
+        <dependency>
+            <groupId>org.assertj</groupId>
+            <artifactId>assertj-swing-testng</artifactId>
+            <version>3.6.0</version>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.assertj</groupId>
+            <artifactId>assertj-swing-aut</artifactId>
```

Comment:
Can you check if there's a typo? I can't find this dependency (assertj-swing-aut).

Reply:
Sorry I forgot to mention this. https://github.com/assertj/assertj-examples/issues/41#issuecomment-435342612 .
I added this to my local maven repository using the same commands that we use for adding sync client https://github.com/odk-x/suitcase/blob/master/dependencies/mvn_local_installs.